src/rmp/test/gcd_restructure.ok

```diff
@@ -20,7 +20,11 @@ Found 69 end points for restructure
 Found 1290 pins in extracted logic.
 Found 422 instances for restructuring.
 [INFO RMP-0002] Blif writer successfully dumped file with 422 instances.
-Reading ABC log results/abc_rcon.log0.
+Warning: Detected 2 multi-output gates (for example, "FA_X1").
+Fraiging part    1  (out of    1)  PI =    89. PO =    52. And =   1073. Lev =   20.                                                                                          ->                                                                             -------->                                                                      --------------->                                                               ---------------------->                                                        ----------------------------->                                                 ------------------------------------>                                          ------------------------------------------->                                   -------------------------------------------------->                                                                                                           Warning: Detected 2 multi-output gates (for example, "FA_X1").
```

Comment:
What is Fraiging  ?

Reply:
"Functionally-Reduced And-Inverter Graph". It's an ABC term.

Reply:
Those messages are printed by abc with no option to turn them off from the tcl api. I think once I start using the c++ api I can turn them off.